docs/src/pages/index.mdx

```diff
@@ -39,8 +39,15 @@ import { Tab, Tabs } from "nextra-theme-docs";
 
 ```bash copy
 UPLOADTHING_SECRET=... # A secret key for your app (starts with sk_live_)
+UPLOADTHING_URL=https://yourapp.com # The URL to your app
```

Comment:
I almost want to not include this and have the callout say that you can specify `UPLOADTHING_URL` if you are not deploying on Vercel. Also this block probably doesn't need the copy button.

Reply:
Agree!